sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -247,6 +248,32 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
+@pytest.mark.parametrize("linear_model", [Lasso])
+def test_model_pipeline_same_as_normalize_true(linear_model):
+    # Test that linear model set with normalize set to True is doing the same
+    # as the same linear model preceeded by StandardScaler in the pipeline and
+    # with normalize set to False
+    X, y, X_test, y_test = build_dataset(n_features=10)
```

Comment:
So here the idea would be that you check with `X` and `X_sparse=sparse.csr_matrix(X)` and it should give the same results